### PR TITLE
Fix #7313, formatting

### LINKED ISSUE #7313: Climbing into disposals doesn't update mob light emission
Light emitted by mob stays at tile they were in before the crawled in disposals, until updated.

### PR DESCRIPTION
**Pull request by ghost**
- Use forceMove() instead of manually adjusting loc, fixes #7313
- A mob eye adjustment was removed along the way, since it's also handled by forceMove()

Other changes : 
- Tons of formatting, removing a bunch of useless "else return" segments (implicit in DM language, if not in all major coding languages), macros, \the, bit of comment formatting
- Squash item check slightly. No need to check that it isn't a mob before checking if it's an item, since it's an exception before the rest of the code

### CODE REVIEW

**Comment by clusterfack** [comment]
I have to ask, were you born with the clumsy gene or is breaking something every PR something you do intentionally?

**Comment by ghost** [comment]
Wait, what ?

**Comment by clusterfack** [comment]
You call it "formatting" I call it removing the check that prevents you from dropping mobs into the disposals when you are restrained or cannot move. And there were several ways you could have formatted this that would both look better AND not break things.

**Comment by 9600bauds** [comment]
I mean it's fun to meme on Dylan but let's be more constructive
Like an explanation of the better formatting ways

**Comment by ghost** [comment]
Oh wait, I'm seeing it now

I guess it would make sense to inverse the checks instead, I'll take a look when I can code again, which is going to be a bit

File: code/modules/recycling/disposal.dm [hunk 1]
```diff
@@ -439,66 +439,62 @@
...
-			return
+		target.visible_message("[target] starts climbing into \the [src].", "You start climbing into \the [src].")
+
 	else
-		if(isanimal(user))
-			return // animals cannot put mobs other than themselves into disposal
 
-		if(!user.restrained() && user.canmove)
-			if(target.locked_to)
-				return
```
Comment (Intigracy) [inline code/modules/recycling/disposal.dm:467]: Before: If you couldn't move but weren't locked_to, **someone else** could disposal you.

Now: You can't be disposaled period if you can't move.

File: code/modules/recycling/disposal.dm [hunk 2]
```diff
@@ -439,66 +439,62 @@
...
+		attackby(dropping, user)
 		return
 
 	var/locHolder = dropping.loc
 	var/mob/target = dropping
 
+	//Our target, now confirmed to be a mob, can't move or is locked to something, same thing
+	if(!target.canmove || target.locked_to)
+		return
+
```
Comment (Intigracy) [inline code/modules/recycling/disposal.dm:465]: Compare here with below.

File: code/modules/recycling/disposal.dm [hunk 3]
```diff
@@ -439,66 +439,62 @@
...
-			user.visible_message("[user] starts stuffing [target] into the [src].", "You start stuffing [target] into the [src].")
-		else
-			return
+		user.visible_message("[user] starts stuffing \the [target] into \the [src].", "You start stuffing \the [target] into \the [src].")
 
 	if(!do_after(user, src, 20))
 		return
 
+	if(user.restrained() || !user.canmove || user.locked_to)
+		return
```
Comment (Intigracy) [inline code/modules/recycling/disposal.dm]: Same thing here.

File: code/modules/recycling/disposal.dm [hunk 4]
```diff
@@ -439,66 +439,62 @@
...
+
+	if(!target.canmove || target.locked_to)
+		return
```
Comment (Intigracy) [inline code/modules/recycling/disposal.dm:483]: Same thing here.

File: code/modules/recycling/disposal.dm (hunk 1 above)
Comment (ghost) [inline code/modules/recycling/disposal.dm:467]: Fixed

**Pull request merged**